modules/opensearch/src/OpenSearchTingObject.php

```diff
@@ -224,7 +410,20 @@ public function getReplacedBy() {
    * {@inheritdoc}
    */
   public function getIsbn() {
-    return $this->getRecordEntry('dc:identifier', 'dkdcplus:ISBN');
+    $isbn = array();
+
+    // Nothing to do.
+    if (empty($this->openSearchObject->record['dc:identifier']['dkdcplus:ISBN'])) {
+      return $isbn;
+    }
+
+    // Get ISBN numbers.
+    $isbn = $this->openSearchObject->record['dc:identifier']['dkdcplus:ISBN'];
+    foreach ($isbn as $k => $number) {
+      $isbn[$k] = str_replace(array(' ', '-'), '', $number);
+    }
+    rsort($isbn);
+    return $isbn;
```

Comment:
This method is changed from returning a string to an array of strings. Is that intentional?

I just noticed the change. I have not been able to extract what the calling code expects.

Reply:
getRecordEntry() will also return an array, so we should be good.